bindings/python/test/trajectory/test_orbit.py

```diff
@@ -98,6 +98,8 @@ def test_get_pass_with_revolution_number(self, orbit: Orbit):
         assert isinstance(pass_, Pass)
         assert pass_.is_defined()
 
+        assert orbit.get_pass_with_revolution_number(2, Duration.minutes(5.0)) is not None
```

Comment:
```suggestion
        assert orbit.get_pass_with_revolution_number(2, Duration.minutes(10.0)) is not None
```
Changing the duration to 5 mins here is confusing, why not leave it at 10 as it is with the default